src/main/ruby/core/truffle/interop.rb

```diff
@@ -53,6 +62,11 @@ def self.object_key_info(object, name)
         flags << :writable if writable
         flags << :existing if readable || writable
       end
+      if name.to_s[0] == '@' && object.instance_variable_defined?(name)
```

Comment:
This check should be earlier, no?
I don't think we want to `object.respond_to?(name)` with name starting with `@`.

Reply:
Done.